flexible-event/privacy.test.ts

```diff
@@ -1,11 +1,12 @@
+import { strict as assert } from 'assert'
```

Comment:
Should we use `import assert from 'node:assert';`

Reply:
How does that work with `strict`?